AAP-17700: Updates to Builder documentation (#957)
ansible/aap-docs: downstream/modules/builder/ref-definition-file-images.adoc

[id="ref-definition-file-image"]

= Images

> The `images` section of the definition file is a dictionary used to define the base image. Verification of signed container images is supported with the `podman` container runtime.

Reviewer: ```suggestion
The `images` section of the definition file identifies the base image. Verification of signed container images is supported with the `podman` container runtime.
```
